.github/workflows/cluster_endtoend_onlineddl_vrepl.yml

```diff
@@ -16,7 +16,7 @@ env:
 jobs:
   build:
     name: Run endtoend tests on Cluster (onlineddl_vrepl)
-    runs-on: gh-hosted-runners-16cores-1
+    runs-on: ${{ github.repository_owner == 'vitessio' && 'gh-hosted-runners-16cores-1' || 'ubuntu-latest' }}
```

Comment:
I thunk we should pin a to a specific known working ubuntu version and not `latest`. That we do that in other jobs in a mistake imho, we explicitly pinned it in the past to avoid having to scramble when it gets updated.

Reply:
I'm okay with that, but I think that should probably go into a separate PR, right? 🤔

Reply:
Agree with @dbussink on this. we should not be overriding the vitessio org runner with `ubuntu-latest`, it should be pinned to the actual latest version.

Reply:
Thanks for the heads up. Will upgrade this tomorrow